massa-bootstrap/src/client_binder.rs

```diff
@@ -101,7 +129,30 @@ impl BootstrapClientBinder {
 
         // save prev sig
         self.prev_sig = Some(sig);
+        self.sended_message = None;
 
         Ok(message)
     }
+
+    #[allow(dead_code)]
+    /// Send a message to the bootstrap server
+    pub async fn send(&mut self, msg: BootstrapMessage) -> Result<(), BootstrapError> {
```

Comment:
You're using `BootstrapMessage` as input for the client `send`, but when I read the enum like it is made for the server only. Should the client not have a different type for its messages given he won't send the same info as the server or am I missing something? AFAIK client needs to send info such as `StartBootstrap` `ContinueFrom` etc.

Reply:
For the moment it's server only but with #2577 it will be client and server. Splitting could be possible but in my mind it loose the """""genericity""""" (I don't know why it's like a feeling) and in the `Message` of the Network all are in the same enum two.

I don't really see a really good reason to keep it group or split it so I have follow what is already done.

Moreover I think we could change it in the other PR because if I split in this one the Client enum will have no variant and it will be a pain to write the tests then.

Reply:
Ok, IMHO it makes more sens to split the messages.

Reply:
Let's see it in #2577